packages/ui-kit/src/components/ThemeProvider/ThemeProvider.types.ts

```diff
@@ -0,0 +1,44 @@
+import type { ChartConfiguration } from 'chart.js'
+import React from 'react'
+import type { ThemeTokenProps } from '../../themes/theme.types'
+
+export type ThemeContextProps = {
+  theme?: ThemeTokenProps
+
+  /**
+   * Configuration function for default `Chart.js` settings.
+   * It allows defining or overriding chart configurations globally.
+   */
+  globalChartConfigProps?: (config: ChartConfiguration<ChartVariant>) => ChartConfiguration<ChartVariant>
```

Comment:
Now it's [options](https://www.chartjs.org/docs/latest/general/options.html)-based global configuration instead of `Chart.defaults`-based as it was before. So, it's scoped within `ThemeProvider` context.

Reply:
Great 👍